pkg/blobaccess/forward.go

```diff
@@ -0,0 +1,103 @@
+package blobaccess
+
+import (
+	"io"
+
+	"github.com/mandelsoft/vfs/pkg/vfs"
+	"github.com/opencontainers/go-digest"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess/bpi"
+	"github.com/open-component-model/ocm/pkg/blobaccess/file"
+	"github.com/open-component-model/ocm/pkg/blobaccess/standard"
+)
+
+///////////
+// Standard
+///////////
+
+func DataAccessForReaderFunction(reader func() (io.ReadCloser, error), origin string) bpi.DataAccess {
+	return standard.DataAccessForReaderFunction(reader, origin)
+}
+
+// DataAccessForBytes wraps a bytes slice into a DataAccess.
+// Deprecated: used DataAccessForData.
+func DataAccessForBytes(data []byte, origin ...string) bpi.DataSource {
+	return standard.DataAccessForData(data, origin...)
+}
+
+// DataAccessForData wraps a bytes slice into a DataAccess.
+func DataAccessForData(data []byte, origin ...string) bpi.DataSource {
+	return standard.DataAccessForData(data, origin...)
+}
+
+func DataAccessForString(data string, origin ...string) bpi.DataSource {
+	return standard.DataAccessForString(data, origin...)
+}
+
+// ForString wraps a string into a BlobAccess, which does not need a close.
+func ForString(mime string, data string) BlobAccess {
```

Comment:
```suggestion
func BlobAccessForString(mime string, data string) BlobAccess {
```

Reply:
Blobaccess creation in a blockaccess package is for generally ForXXX. to avoid repetition of package name.
You can write now `blobaccess.ForXXX(...)`.